cfme/cloud/tenant.py

```diff
@@ -68,10 +72,9 @@ def wait_for_disappear(self, timeout=300):
         except TimedOutError:
             logger.error('Timed out waiting for tenant to disappear, continuing')
 
-    # def wait_for(self, timeout=600):
-    #     return wait_for(self.exists, fail_condition=False, timeout=timeout,
-    #                     message='Wait for cloud tenant to appear', delay=10,
-    #                     fail_func=sel.refresh)
+    def wait_for_appear(self, timeout=600):
+        return wait_for(self.exists, timeout=timeout, message='Wait for cloud tenant to appear',
+                        delay=10, fail_func=sel.refresh)
```

Comment:
Discussed offline - the 'All' destination needs a resetter method for this to work as I was pretending it did.